apps/summon-app/src/main.tsx

```diff
@@ -1,20 +1,27 @@
-import { HausConnectProvider } from '@daohaus/daohaus-connect-feature';
-import { HausThemeProvider } from '@daohaus/ui';
 import { StrictMode } from 'react';
-import * as ReactDOM from 'react-dom';
-import { BrowserRouter } from 'react-router-dom';
+import { createRoot } from 'react-dom/client';
+import { HashRouter } from 'react-router-dom';
+
+import { HausThemeProvider } from '@daohaus/ui';
+import { HausConnectProvider } from '@daohaus/daohaus-connect-feature';
 
 import App from './app/App';
 
-ReactDOM.render(
+const container = document.getElementById('root');
+
+//reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis
+// THis is how react wants to render the app.
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const root = createRoot(container!);
```

Comment:
Same

Reply:
Yeah, I didn't actually get any problems when implementing it. Was just following the docs as they had recommended it if we're using TS, and our linter doesn't allow type assertions. 

Figured I would follow the docs on the small chance that there's some sort of downstream linting headache that emerges out of the blue.

Reply:
Yeah, I noticed that. 
If you do Create React App with Typescript it does it the way I posted. Odd that it isn't consistent with the docs.